src/services/download/s3.rs

```diff
@@ -144,3 +144,224 @@ impl S3Downloader {
         .collect()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::sources::{CommonSourceConfig, DirectoryLayoutType, SourceId};
+    use crate::test;
+    use crate::types::ObjectType;
+
+    use super::*;
+    use sha1::{Digest as _, Sha1};
+
+    const S3_BUCKET: &str = "symbolicator-test";
+
+    fn s3_source_key() -> Option<S3SourceKey> {
+        let access_key = std::env::var("AWS_ACCESS_KEY_ID").ok()?;
+        let secret_key = std::env::var("AWS_SECRET_ACCESS_KEY").ok()?;
```

Comment:
```suggestion
        let access_key = std::env::var("SENTRY_SYMBOLICATOR_AWS_ACCESS_KEY_ID").ok()?;
        let secret_key = std::env::var("SENTRY_SYMBOLICATOR_AWS_SECRET_ACCESS_KEY").ok()?;
```

Reply:
This was an intentional change, but I've reverted it now.